movement/utils/vector.py

```diff
@@ -6,6 +6,72 @@
 from movement.utils.logging import log_error
 
 
+def magnitude(data: xr.DataArray) -> xr.DataArray:
+    """Compute the magnitude in space.
+
+    The magnitude is computed as the Euclidean norm of a vector
+    with spatial components ``x`` and ``y`` in Cartesian coordinates.
+    If the input data contains polar coordinates, the magnitude
+    is the same as the radial distance ``rho``.
+
+    Parameters
+    ----------
+    data : xarray.DataArray
+        The input data containing either ``space`` or ``space_pol``
+        as a dimension.
+
+    Returns
+    -------
+    xarray.DataArray
+        An xarray DataArray representing the magnitude of the vector
+        in space. The output has no spatial dimension.
+
+
+    """
+    if "space" in data.dims:
+        _validate_dimension_coordinates(data, {"space": ["x", "y"]})
+        return xr.apply_ufunc(
+            np.linalg.norm,
+            data,
+            input_core_dims=[["space"]],
+            kwargs={"axis": -1},
+        )
+    elif "space_pol" in data.dims:
+        _validate_dimension_coordinates(data, {"space_pol": ["rho", "phi"]})
+        return data.sel(space_pol="rho", drop=True)
+    else:
+        raise log_error(
+            ValueError,
+            "Input data must contain either 'space' or 'space_pol' "
+            "as dimensions.",
+        )
+
+
+def normalize(data: xr.DataArray) -> xr.DataArray:
```

Comment:
Quick quesiton @sfmig:

What do you think about `normalize_to_unit` or `convert_to_unit`? I feel like "compute" implies the production of a new quantity, while here I want to communicate that this functions doesn't produce something new but instead converts to a normalised format.

Also, do you think the inclusion of `vector` in the function name is necessary? I thought it might be superfluous because the whole module is named `vector.py`, but I might be overthinking this.

For context, I've decided to go with your `compute_norm` suggestion for the other function.

Reply:
> What do you think about normalize_to_unit or convert_to_unit? 

I think `convert_to_unit` is clear and explicit, nice one! I don't feel strongly about "compute" but I see your point. 

> Also, do you think the inclusion of vector in the function name is necessary?

Yes I think you are right, it feels a bit redundant.

> I might be overthinking this.

I feel ya, it's kinda tricky 😅  (was there ~24h ago)